ogr/abstract.py

```diff
@@ -806,6 +806,22 @@ def project_create(
         """
         raise NotImplementedError()
 
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_patter: str = None,
```

Comment:
Probably a typo.

```suggestion
        search_pattern: str = None,
```

Would be nice to mention that the pattern is for project name -- I would cover it in the attribute name or in a docsring.